sg.config.yaml

```
@@ -650,6 +650,9 @@ commands:
       - monitoring
     continueWatchOnExit: true
 
+  env:
```

As it is it's a bit hard to make sense of if you don't know the commit message.

How about renaming and adding a comment?

```suggestion
  # This will execute `env`, a utility to print the process environment. Can be used to debug which global vars `sg` uses.
  debug-env:
```